### PR TITLE
Don't send http errors to Sentry.

### PR DESCRIPTION
**Pull request by matthew-andrews**
@phamann

### CODE REVIEW

**Comment by matthew-andrews** [comment]
https://app.getsentry.com/nextftcom/ft-next-personalised-feed-api/group/72242426/

**Comment by matthew-andrews** [comment]
https://app.getsentry.com/nextftcom/ft-next-personalised-feed-api/group/72242426/

File: server/controllers/v1/feed.js [hunk 1]
```diff
@@ -104,6 +105,14 @@ module.exports = function (req, res, next) {
 
 				res.json(responseObject);
 			})
+			.catch(function(err) {
+				if (err.name === fetchres.BadServerResponseError.name
+					|| err.name.indexOf('timeout') > -1) {
+					res.sendStatus(504);
```
Comment (wheresrhys) [inline server/controllers/v1/feed.js]: log?

Comment (matthew-andrews) [inline server/controllers/v1/feed.js]: it will be logged in fastly i believe, what additional debug do we need?

**Pull request merged**